src/constants/index.js

```diff
@@ -18,6 +18,8 @@ export const CLEAR_USER_MSGS = 'CLEAR_USER_MSGS'
 export const COMPOSE_CREATE_VM = 'COMPOSE_CREATE_VM'
 export const CREATE_DISK_FOR_VM = 'CREATE_DISK_FOR_VM'
 export const CREATE_VM = 'CREATE_VM'
+export const DEFAULT_ENGINE_OPTION_VALUE = '_'
+export const DEFAULT_ARCH = '_'
```

Comment:
I think `DEFAULT` still fits since almost everything specifics a "the value if the one you asked for isn't found" as a default.  I'll think about it bit more.

Reply:
Maybe `UNKNOW_ARCH` does make sense.